static/js/services/delete-docs.js

```diff
@@ -75,22 +108,33 @@ var _ = require('underscore');
         docs.forEach(function(doc) {
           doc._deleted = true;
         });
+        var parents = [];
         return $q.all(docs.map(function(doc) {
           return getParent(doc)
             .then(function(parent) {
               if (parent) {
-                docs.push(parent);
+                parents.push(parent);
               }
             });
           }))
           .then(function() {
-            return checkForDuplicates(docs);
-          })
-          .then(function() {
-            return minifyLineage(docs);
-          })
-          .then(function() {
-            return DB().bulkDocs(docs);
+            var docsWithParents = docs.concat(parents);
+            checkForDuplicates(docsWithParents);
+
+            if (!Session.isAdmin()) {
```

Comment:
Is this the right check to make? Are national admins (who are online users) also counted in `isAdmin`?

NB: there is this ticket: https://github.com/medic/medic-webapp/issues/3960

Which talks about finding a better way to determine online vs. offline users. You're welcome to take a crack a that :-)

Reply:
Yeah it felt a bit wrong using this check, happy to take a crack at that other issue to get to something less hard-coded